tests/before_build/steps/events.py

```diff
@@ -144,6 +144,24 @@ def valid_external_url(field, value, error):
     }
 }
 
+banner_schema = {
+    'image_url': {
+        'validator': valid_external_url,
+        'required': False
+    }
+}
+
+featured_video_schema = {
```

Comment:
Change this to `youtube_url_schema`

Reply:
okay, changed. Waiting for checks to pass